chapters/classes.tex

```diff
@@ -1500,6 +1500,12 @@ \subsection{Real Type}\label{real-type}
 \index{unbounded@\robustinline{unbounded}!attribute of \robustinline{Real}}%
 \index{stateSelect@\robustinline{stateSelect}!attribute of \robustinline{Real}}
 
+The default \lstinline!min!- and \lstinline!max!-attributes do not constrain the value.
```

Comment:
What are _default_ attributes? It can't be default values since there are no values assigned.

Reply:
I have tried to rewrite that, but still without introducing a special `+Inf` symbol.
If that fails we can just skip the last two commits.